pkg/proxy/namespace_round_tripper.go

```
@@ -48,9 +48,11 @@ func (m *MappingManager) loadMappings(filePath string) error {
 	if err != nil {
 		return err
 	}
-	defer file.Close()
+	defer func() {
+		_ = file.Close()
```

Don't ignore the error. Log it.